### ISSUE TITLE
Accessibility Issue #1: Screen Reader Compatibility

### ISSUE THREAD

**Issue by LAYAIraq** [LAYAIraq/LAYA-Iraq-Frontend#73]
In GitLab by @vieregni on Sep 6, 2021, 14:06

Every component of LAYA should be selectable via keyboard and be therefore accessible for screen readers.
 
- [ ] Course List
- [ ] Profile
- [ ] Login
- [ ] Plugin - SCMC
- [ ] Plugin - Dialog
- [ ] Plugin - Feedback
- [ ] Plugin - Relate
- [ ] Plugin - Sort
- [ ] Plugin - DragnDrop

Helpful guides:
- https://medium.com/@emilymears/getting-started-with-web-accessibility-in-vue-17e2c4ea0842
- https://www.freecodecamp.org/news/build-accessible-vue-applications/
- https://www.deque.com/blog/a-vue-of-accessibility-tips-and-tricks-to-make-your-application-accessible/

Test via NVDA (Windows) or Orca (Linux).
